### DISCUSSION TITLE
Why are Pages and Razor Configuration Included in an API-Only Template?

### DISCUSSION THREAD

**Discussion by tzdevelopteam** [jasontaylordev/CleanArchitecture#1282] [General]
**Is your question related to a problem? Please describe.**  
While using the clean architecture template for building an API-only project, I noticed that the `Pages` directory and Razor configuration are included by default. Since Razor Pages are typically used for server-rendered web UI and not for APIs, I'm wondering about the necessity of these components in an API-centric setup.

**Describe the question or clarification you need**  
Could you clarify why `Pages` and Razor configuration are included in the template, even if it is intended to be used primarily for APIs? Are there specific scenarios or design principles that justify their presence in an API-focused architecture?

**Additional context**  
If these components are not essential for an API-only setup, would it make sense to create a streamlined version of the template without Razor dependencies? This could reduce unnecessary complexity and improve performance for API-first projects.

**Comment by jasontaylordev** [comment]
Hi @tzdevelopteam. Thanks for reporting this issue; this is a bug, and I've prepared a bug fix. Can you advise what you mean by Razor configuration?

**Reply by tzdevelopteam** [reply]
app.MapRazorPages(); For example

**Comment by jasontaylordev** [comment]
This is now fixed
